manifest.yml

```diff
@@ -0,0 +1,16 @@
+---
+applications:
+- name: wavefront-proxy-service-broker
+  instances: 1
+  memory: 1G
+  path: target/wavefront-cloudfoundry-broker-0.0.1-SNAPSHOT.jar
+  buildpack: https://github.com/cloudfoundry/java-buildpack.git
+  # change the route domain name accordingly
+  route:
+  - route: wavefront-proxy-service-broker.apps.wavefront.io
+  env:
+    SECURITY_USER_NAME: admin
```

Comment:
This is just a default username/password that is used for dev testing when we manually push this as an app. The CF framework auto generates these when registering a service broker as part of the tile.